app.js

```diff
@@ -66,208 +31,222 @@ const logger = winston.createLogger({
     ]
 });
 
-
 // define jobQueue handler
 const filePreviewQueue = kue.createQueue(jobqueueConfig);
-filePreviewQueue.process('filePreview', function (job, done) {
+filePreviewQueue.process('filePreview', function(job, done) {
     logger.debug('processing job: ' + job.id);
     handleFilePreview(
         job.data.options,
-        job.data.download_url,
-        job.data.signed_s3_url,
+        job.data.downloadUrl,
+        job.data.signedS3Url,
         done
     );
 });
 
-
 // let's create the app
 const app = express();
-app.use(bodyParser.json())
+app.use(bodyParser.json());
 
 app.listen(appConfig.port || 3000, function() {
-    logger.info(util.format('Example app listening on port %s!', (appConfig.port || '3000')));
+    logger.info(
+        util.format(
+            'Example app listening on port %s!',
+            appConfig.port || '3000'
+        )
+    );
 });
 
-app.post('/filepreview', passport.authenticate('basic', { session: false }), function (req, res) {
-    if (!req.body.callback_url) return res.status(422).send({error: 'request must contain callback_url'})
-    if (!req.body.download_url) return res.status(422).send({error: 'request must contain download_url'})
-    if (!req.body.signed_s3_url) return res.status(422).send({error: 'request must contain signed_s3_url'})
-
-    const callback_url  = req.body.callback_url;
-    const download_url  = req.body.download_url;
-    const signed_s3_url = req.body.signed_s3_url;
+app.post(
+    '/filepreview',
+    //passport.authenticate('basic', { session: false }),
+    function(req, res) {
+        if (!req.body.callbackUrl)
+            return res
+                .status(422)
+                .send({ error: 'request must contain callbackUrl' });
```

Comment:
there are several http-error packages existing for that use case. validation can be done in an before hook instead or separate this at list into a funtion